app/Http/Requests/PackageUpdateRequest.php

```diff
@@ -28,7 +28,7 @@ public function rules()
             'price'                => 'required|numeric',
```

Comment:
remove this file and use package store request as pacake update request

Reply:
this file can't be deleted because the package name is unique